src/media/js/main.js

```diff
@@ -230,6 +202,8 @@ function(_) {
             $('#site-nav').after(nunjucks.env.render('incompatible.html'));
         }
 
+        update_banner.showIfneeded();
```

Comment:
Can't we also do this within the update_banner module?

Camelcase: showIfNeeded